src/main/java/io/seqera/migtool/Helper.java

```diff
@@ -216,6 +216,8 @@ static public String driverFromUrl(String url) {
             return "org.h2.Driver";
         if( "sqlite".equals(dialect))
             return "org.sqlite.JDBC";
+        if( "postgresql".equals(dialect))
```

Comment:
In my proposed implementation, we got rid of these hardcoded strings, by using an enum to get all the behaviour related to either Driver and Dialect abstracted. That way, we can modify and test it easily.

Reply:
it's irrelevant in this context

Reply:
Since I needed to add the support for Postgres, it was way easier to me to refactor it than to leave it as it is, since it is just not very friendly to understand: methods doing more than one thing, hardcoded strings all over, using Strings instead of types, related code split into several classes... This is why I refactored it, so it would be easier to extend.

Reply:
Again, not relevant in this context. The code is so small that a constant or enum adds not value. It much simple grep/search for a specific string literal. A refactor can be considered in a separate issue.